crates/astria-conductor/src/executor/mod.rs

```diff
@@ -128,12 +133,29 @@ impl Executor {
             .await
             .wrap_err("executor failed to get commitment state")?;
 
+        // The `executable_block_height` is the height of the fist sequencer block
+        // that can be executed on top of the rollup state. The `init_sequencer_height`
+        // is set when the rollup is first created and let's us know that this
+        // rollup's genesis block is in block K of the sequencer chain.
```

Comment:
```suggestion
        // rollup's first block is in block K of the sequencer chain.
```

genesis block is defined in the genesis file, it's the rollup block 1 that's in a sequencer block (as it's the first executable block)